packages/client/hmi-client/src/components/workflow/tera-simulation-workflow.vue

```diff
@@ -88,6 +90,18 @@ const testOperation: Operation = {
 	isRunnable: true
 };
 
+function appendOutputPort(nodeId: string, outputPortData: WorkflowPort) {
+	// Find node and assign outport data to its output port
+	const node = nodes.value[nodes.value.findIndex(({ id }) => id === nodeId)];
+	node.outputs[node.outputs.length - 1] = outputPortData;
+
+	// Create new output port
+	node.outputs.push({
+		id: node.outputs.length.toString(),
+		type: outputPortData.type
+	});
+}
```

Comment:
I am not sure if this is the pattern we want - that there is always an "empty" output-port that cannot be used.

I think it is good for now but we should have a bit of discussion around it. 
CC @echl @mwdchang @Tom-Szendrey